richardaspinall/poker-react-node
# refactor: poker table into lobby

### Description



This PR refactors how we create and manage poker tables. Instead of the PokerTable class managing the creation of tables, we now have a PokerLobby that looks after instances of PokerTable

### How to test



1. Open two windows and the developer tools
2. Click on a seat on each
3. Note that players sit

### Task


https://app.clickup.com/t/86cu9ten0
CU-86cu9ten0

### Checklist
